Tests/test_image_transform.py

```diff
@@ -16,6 +16,8 @@ def test_sanity(self):
 
         transform = ImageTransform.AffineTransform(seq[:6])
         im.transform((100, 100), transform)
+        transform = ImageTransform.PerspectiveTransform(seq[:8])
+        im.transform((100, 100), transform)
```

Comment:
Can we add some correctness tests rather than just a "didn't crash" test?

Reply:
Ok, done